### PR TITLE
[ADD] l10n_kz: add accounting for Kazakhstan

### PR DESCRIPTION
**Pull request by aliyatastemirova**
[ADD] l10n_kz: add l10n_kz accounting localization
Adds a Chart of Accounts, taxes, and a demo company for Kazakhstan.
- The Chart of Accounts is added according to the official documentation:
  https://adilet.zan.kz/rus/docs/V070004771_
- The Tax report is added according to this documentation:
  https://adilet.zan.kz/rus/docs/V14E0010156

task-2693102

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_kz/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```
Comment (william-andre) [inline addons/l10n_kz/__manifest__.py:1]: Don't keep the icons, let the designers deal with it

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Comment by aliyatastemirova** [comment]
@william-andre should be ok now, sorry

**Comment by william-andre** [comment]
@robodoo r+

**Comment by yelizariev** [comment]
Thank you!
image

**Pull request closed without merging**